app/src/main/java/com/example/marvelapp/di/ViewModelModule.kt

```diff
@@ -0,0 +1,18 @@
+package com.example.marvelapp.di
+
+import com.example.marvelapp.mvvm.model.CharacterListModel
+import com.example.marvelapp.mvvm.viewmodel.CharacterListViewModel
+import com.example.marvelapp.mvvm.viewmodel.MainViewModel
+import com.example.marvelapp.mvvm.viewmodel.SplashViewModel
+import org.koin.androidx.viewmodel.dsl.viewModel
+import org.koin.dsl.module
+
+object ViewModelModule {
+    val viewModelModule = module {
+
+        viewModel { SplashViewModel() }
+        viewModel { MainViewModel() }
+        viewModel { CharacterListViewModel(get()) }
+        single { CharacterListModel() }
```

Comment:
1) This should be in a separated module
2) Why it is a single? I think that a factory should be better